apps/collector/collector/event_types.go

```diff
@@ -0,0 +1,20 @@
+package collector
+
+type EventBuffering struct {
+	URL            string `json:"url"`
+	Position       int    `json:"position"`
+	Duration       *int   `json:"duration,omitempty"`
+	StreamDuration *int   `json:"stream_duration,omitempty"`
+	StreamBitrate  *int   `json:"stream_bitrate,omitempty"`
+}
+
+type PlaybackStart struct {
+	// media bitrate, bits/sec (if known)
+	StreamBitrate *int `json:"stream_bitrate,omitempty"`
+	// stream duration, seconds
+	StreamDuration *int `json:"stream_duration,omitempty"`
+	// time before playback started, milliseconds
```

Comment:
these comments make me think that the names of the fields are not clear enough. the name should tell you exactly what the thing is.

also, making some fields be seconds and some be milliseconds is a recipe for mistakes. pick one or the other and make all times be that.

Reply:
The comments are auto-generated from the openapi spec. Agree on the units.